lerna.json

```diff
@@ -1,9 +1,9 @@
 {
-  "lerna": "2.1.2",
+  "lerna": "3.15.0",
```

Comment:
The lerna key looks like it can be removed, it is now longer listed in [the lerna.json documentation](https://github.com/lerna/lerna#lernajson).